### ISSUE TITLE
comment on first line of templates affects rendering

### ISSUE THREAD

**Issue by BenoitBotton** [Tresjs/tres#528]
### Describe the bug

a select is meant to swap between sets of objects in the scene.
when the code has a comment on the first line of the template, objects are added to the scene, the old ones are not removed. 

### Reproduction

https://stackblitz.com/edit/tresjs-basic-hmkhgf?file=src%2Fcomponents%2FDonut.vue

### Steps to reproduce

run the stackblitz.
change the select to 'donuts 1', a second set of donuts is added to the scene.

delete the comment in the template (Donut.vue) and refresh the view
this time when switching using the select, the scene objects are changed, not added

### Used Package Manager

npm

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/Tresjs/tres/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guidelines](https://github.com/Tresjs/tres/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://tresjs.org/guide).
- [X] Check that there isn't [already an issue](https://github.com/tresjs/tres/issues) that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by garrlker** [comment]
Just tested the 4.0.2 release in your reproduction, it looks like #717 did not fix it. We'll have to do more digging into how v-for is working behind the scenes

**Comment by alvarosabu** [comment]
@Tinoooo I have the feeling your latest PR might have solved this?

**Comment by Tinoooo** [comment]
Yes it is fixed by #1283 . When using the latest version in the reproduction the bug does not occur:
https://stackblitz.com/edit/tresjs-basic-zbtmu1nn
